src/app.js

```diff
@@ -1,8 +1,13 @@
+require('dotenv').config({path:'./env'})
+require("express-async-errors");
 const express = require('express');
 const app = express();
 const cors = require('cors')
 const favicon = require('express-favicon');
 const logger = require('morgan');
+const express = require("express");
+
+const authenticateUser = require('./middleware/authentication');
```

Comment:
this line is not used in this file and the path doesn't take you to the right file (shows as error on my end)

Reply:
I made an app.use statement and am going to test it now before pushing..